### PR TITLE
#474 init location service

### PR DESCRIPTION
**Pull request by nguyenvanhadncntt**

### CODE REVIEW

File: location/pom.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        <java.version>17</java.version>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <springdoc-openapi-starter-webmvc-ui.version>2.0.2</springdoc-openapi-starter-webmvc-ui.version>
+        <org.lombok.version>1.18.22</org.lombok.version>
+        <org.mapstruct.version>1.4.1.Final</org.mapstruct.version>
+        <spring-cloud.version>2022.0.1</spring-cloud.version>
+        <sonar.organization>nashtech-garage</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.projectKey>nashtech-garage_yas-order</sonar.projectKey>
```
Comment (thiennn) [inline location/pom.xml:26]: nashtech-garage_yas-location

File: location/src/main/java/com/yas/location/config/ServiceUrlConfig.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.yas.location.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "yas.location")
+public record ServiceUrlConfig(String media, String product) {
```
Comment (LEHOANGGLAM) [inline location/src/main/java/com/yas/location/config/ServiceUrlConfig.java:6]: Is location going to communicate with media or product service?

File: location/pom.xml (hunk 1 above)
Comment (nguyenvanhadncntt) [reply location/pom.xml]: Thanks, I have fixed it, please review again.

File: location/src/main/java/com/yas/location/config/ServiceUrlConfig.java (hunk 2 above)
Comment (nguyenvanhadncntt) [reply location/src/main/java/com/yas/location/config/ServiceUrlConfig.java]: Thanks, I have fixed it, please review again.

**Review by LEHOANGGLAM — APPROVED** [review]

**Review by toannguyenk — APPROVED** [review]

**Pull request merged**